src/front-end/typescript/lib/pages/proposal/team-with-us/lib/views/edit-tab-header.tsx

```diff
@@ -99,6 +103,19 @@ const ViewTabHeader: component.base.View<Props> = ({
       <Row>
         <Col xs="12">
           <DescriptionList items={items} />
+          <Link
+            newTab
```

Comment:
Why does this _need_ to open in a new tab?

Reply:
The ticket indicated style and behaviour of the new link for TWU should be consistent with what was in place for SWU, which opens in a new tab. If that is not the case, then I can update, just let me know.

Reply:
Agreed, the work is consistent with the language of the ticket and should the ticket be free from assumption or error then there is no issue. If we know better, or if there's a different approach that deviates away from preference towards better alignment with WCAG then I guess that's where I'm wondering if there's room to ask if it's necessary.

Reply:
@bdolor All of the export pages open in a new tab as it was a requirement from government during initial implementation. It's also worth mentioning that a vendor would lose any unsaved changes if they were editing their proposal and clicked the "Export Proposal" link and it didn't open in a new tab.